cpp/tests/community/egonet_test.cu

```diff
@@ -118,7 +126,7 @@ class Tests_InducedEgo : public ::testing::TestWithParam<InducedEgo_Usecase> {
     ASSERT_TRUE(h_cugraph_ego_edge_offsets[configuration.ego_sources.size()] ==
                 d_ego_edgelist_src.size());
     for (size_t i = 0; i < configuration.ego_sources.size(); i++)
-      ASSERT_TRUE(h_cugraph_ego_edge_offsets[i] < h_cugraph_ego_edge_offsets[i + 1]);
+      ASSERT_TRUE(h_cugraph_ego_edge_offsets[i] <= h_cugraph_ego_edge_offsets[i + 1]);
     auto n_vertices = graph_view.get_number_of_vertices();
     for (size_t i = 0; i < d_ego_edgelist_src.size(); i++) {
       ASSERT_TRUE(h_cugraph_ego_edgelist_src[i] >= 0);
```

Comment:
Better use https://github.com/rapidsai/cugraph/blob/branch-0.19/cpp/tests/utilities/test_utilities.hpp#L134 as this will generate a warning if vertex_t is unsigned.

Reply:
f81a2b4